main.cpp

```diff
@@ -1599,9 +1600,12 @@ string missing_device_string(bool wasRetry) {
         if (settings.bus != -1) {
             snprintf(buf, buf_len, "accessible RP2040 devices in BOOTSEL mode were found found on bus %d.", settings.bus);
         } else {
-            snprintf(buf, buf_len,"accessible RP2040 devices in BOOTSEL mode were found.");
+            snprintf(buf, buf_len, "accessible RP2040 devices in BOOTSEL mode were found.");
         }
     }
+    if (settings.force) {
+        snprintf(buf, buf_len, "\nTo force a device into BOOTSEL mode, make sure the RP2040 is configured to use USB-CDC (USB stdio).");
```

Comment:
I _think_ you need to recalculate `buf` and `buf_len`, otherwise you'll just be overwriting the previous text rather than appending to it?